page-as-post-type-archive.php

```diff
@@ -6,7 +6,7 @@
  * Plugin Name:     Page as post type archive
  * Plugin URI:      https://github.com/ItinerisLtd/page-as-post-type-archive/
  * Description:     Allows you to set a page as the archive of a post type.
- * Version:         0.1.0
+ * Version:         0.1.1
```

Comment:
```suggestion
 * Version:         0.2.2
```
we never updated this for past tags.. whoops